server_installation/topics/operator/keycloak-client-cr.adoc

```diff
@@ -0,0 +1,67 @@
+
+=== KeycloakClient Custom Resource
+
+{project_operator} allows application developers to represent Keycloak Clients as Custom Resources:
+
+.`KeycloakClient` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakClient
+metadata:
+  name: <Keycloak Client name>
+  labels:
+    app: sso
```

Comment:
Is this `app: sso` label used for something later? (e.g. for a search how many Keycloak clients exist in the corresponding namespace)

IOW does it need always to be set to `app: sso`, since its used for something later, the proper function of the operator relies on, or it can be set to arbitrary value? If it should be always set to `app: sso` it should be mentioned IMHO somewhere here.

Reply:
It might be used in the future I believe. At the moment, we apply those labels for two reasons:
- If `app` label is there, OpenShift shows them a bit nicer in the UI (e.g. you have a Route available in the StatefulSet deployment since it knows it's the same application).
- `Keycloak` and `KeycloakRealm` labels are used to find proper installation and realm to inject the client to. At the moment, there's nothing looking for existing Clients but I guess it's very likely we'll have something in the future.